app/controllers/profiles_controller.rb

```diff
@@ -132,10 +132,21 @@ def profile_params
   end
 
   def search_with_search_params
-    @profiles = matching_profiles
+    @profiles = []
+    matching_profiles.each do |profile|
+      @profiles << {
+                      id: profile.id,
+                      fullname: profile.fullname,
+                      iso_languages: profile.iso_languages,
+                      city: profile.city,
+                      willing_to_travel: profile.willing_to_travel,
+                      nonprofit: profile.nonprofit,
+                      main_topic_or_first_topic: profile.main_topic_or_first_topic
+                    }
+    end
 
     # search results aggregated according to certain attributes to display as filters
-    aggs = ProfileGrouper.new(params[:locale], matching_profiles.ids).agg_hash
+    aggs = ProfileGrouper.new(params[:locale], @profiles.map { |hash| hash[:id] }).agg_hash
```

Comment:
Where does the ProfileGrouper come from?

Reply:
This is a new service Andy and I implemented to group the profiles concerning the facet attributes (country, city, state, language). You will find it only in the other branch against which I created this PR.

Reply:
👍